pkg/exchange/bybit/convert.go

```diff
@@ -0,0 +1,53 @@
+package bybit
+
+import (
+	"math"
+
+	"github.com/c9s/bbgo/pkg/exchange/bybit/bybitapi"
+	"github.com/c9s/bbgo/pkg/types"
+)
+
+func toGlobalMarket(m bybitapi.Instrument) types.Market {
+	// sample:
+	//Symbol: BTCUSDT
+	//BaseCoin: BTC
+	//QuoteCoin: USDT
+	//Innovation: 0
+	//Status: Trading
+	//MarginTrading: both
+	//
+	//LotSizeFilter:
+	//{
+	//    BasePrecision: 0.000001
+	//    QuotePrecision: 0.00000001
+	//    MinOrderQty: 0.000048
+	//    MaxOrderQty: 71.73956243
+	//    MinOrderAmt: 1
+	//    MaxOrderAmt: 2000000
+	//}
+	//
+	//PriceFilter:
+	//{
+	//    TickSize: 0.01
+	//}
+	return types.Market{
+		Symbol:          m.Symbol,
+		LocalSymbol:     m.Symbol,
+		PricePrecision:  int(math.Log10(m.LotSizeFilter.QuotePrecision.Float64())),
+		VolumePrecision: int(math.Log10(m.LotSizeFilter.BasePrecision.Float64())),
+		QuoteCurrency:   m.QuoteCoin,
+		BaseCurrency:    m.BaseCoin,
+		MinNotional:     m.LotSizeFilter.MinOrderAmt,
+		MinAmount:       m.LotSizeFilter.MinOrderAmt,
```

Comment:
These two fields have the same value. Does that expect?

Reply:
Please also add unit tests for this convertor.

Reply:
This has been confirmed with @c9s . Some exchanges have a `MIN_NOTIONAL` and declare it in the API response. But in BYBIT there is only a minimum amount limit, so we fill him. This value is used to confirm the minimum order amount when trading

Reply:
Yes